hpfeeds-cif.env

```diff
@@ -2,25 +2,25 @@
 #
 # This can be modified to change the default setup of the unattended installation
 
-HPFEEDS_HOST='hpfeeds'
+HPFEEDS_HOST=hpfeeds
```

Comment:
Should this default to `hpfeeds3`?

Reply:
env file has been updated to reflect this change.